docs/using-cinder-csi-plugin.md

```diff
@@ -1,3 +1,44 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [CSI Cinder driver](#csi-cinder-driver)
+  - [Kubernetes](#kubernetes)
+    - [Compatibility](#compatibility)
+    - [Requirements](#requirements)
+    - [Example local-up-cluster.sh](#example-local-up-clustersh)
+    - [Deploy](#deploy)
+      - [Using the Helm chart](#using-the-helm-chart)
+      - [Using the manifests](#using-the-manifests)
+    - [Example Nginx application usage](#example-nginx-application-usage)
+    - [Enable Topology-aware dynamic provisioning for Cinder Volumes](#enable-topology-aware-dynamic-provisioning-for-cinder-volumes)
+    - [Example Snapshot Create and Restore](#example-snapshot-create-and-restore)
+    - [Example: Raw Block Volume](#example-raw-block-volume)
+    - [Volume Expansion](#volume-expansion)
+      - [Rescan block device geometry on in-use volume resize](#rescan-block-device-geometry-on-in-use-volume-resize)
+    - [Inline Volumes](#inline-volumes)
+    - [Volume Cloning](#volume-cloning)
+  - [Running Sanity Tests](#running-sanity-tests)
+  - [Using CSC tool](#using-csc-tool)
+    - [Test using csc](#test-using-csc)
+    - [Start Cinder driver](#start-cinder-driver)
+      - [Get plugin info](#get-plugin-info)
```

Comment:
I think all the following section names should be move to upper level (same with 'Start Cinder driver')

Reply:
originally doc has problem...modified